### DISCUSSION TITLE
Git indicators

### DISCUSSION THREAD

**Discussion by mzulfanw** [zed-industries/zed#12433] [Feature Requests]
how to toggle again to the previous ui ? 
![Screenshot 2024-05-29 at 22 25 59](https://github.com/zed-industries/zed/assets/60052964/b3157317-3324-41b5-85cb-25f3fdb4ba10)

**Comment by Moshyfawn** [comment]
You can either press `Esc` or call the `editor: toggle hunk diff` action from the command palette.

There's a recent feature request to add an explicit button to toggle the hunk on and off as well.

- #12404

**Reply by mzulfanw** [reply]
thanks mate @Moshyfawn
